### PR TITLE
Adding fact-store to move over to main dev repo

### PR DESCRIPTION
**Pull request by zmhassan**
## Background
 We are experimenting with building a user feedback driven fact store.
 This will be a service that will utilized to query for finding false predictions. 
 Also will allow users to click which anomaly is not really an anomaly for example.
  
image

I'd like to move this code over to the main development repo.

### CODE REVIEW

**Comment by zmhassan** [comment]
@MichaelClifford @vpavlin @durandom I've moved over the prototype into this repo. Hope you can review this PR and let me know if this is good.

**Comment by durandom** [comment]
@zmhassan could you please re-base this PR on top of current master, so that we get the stickler ci to review as well?

File: fact-store/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea/
```
Comment (durandom) [inline fact-store/.gitignore]: the `.gitignore` should not be here

Comment (zmhassan) [reply fact-store/.gitignore]: Done!

**Comment by zmhassan** [comment]
@durandom It is squashed and clean

**Pull request merged**